src/commands/aem/rde/status.js

```diff
@@ -57,12 +67,53 @@ class StatusCommand extends BaseCommand {
       cli.log(err);
     }
   }
+
+  async printAsJson() {
+    try {
+
+      const status = await this.withCloudSdk((cloudSdkAPI) =>
+        loadAllArtifacts(cloudSdkAPI)
+      );
+
+      const grouped = groupArtifacts(status.items);
+
+      const result = {
+        programId: this._programId,
+        environmentId: this._environmentId,
+        status: status.status,
+      }
```

Comment:
This block should go into the `else` block.

Reply:
This way you will also get the program id and environment id when there is an error, could be useful to see if you misconfigured something there